cypress/testcases/virtualmachines/advanced.spec.ts

```diff
@@ -0,0 +1,60 @@
+import YAML from 'js-yaml'
+
+import { VmsPage } from "@/pageobjects/virtualmachine.po";
+import { LoginPage } from "@/pageobjects/login.po";
+
+
+const vms = new VmsPage();
+const login = new LoginPage();
+
+describe('Create a new VM and add Enable USB tablet option', () => {
+  const VM_NAME = 'test-usb-tablet'
+
+  beforeEach(() => {
+    login.login();
+  });
+
+  it('Step1: Crate VM with Enable usb tablet checked', () => {
+    vms.goToCreate();
+
+    const value = {
+      name: VM_NAME,
+      cpu: '2',
+      memory: '4',
+      image: 'ubuntu-18.04-server-cloudimg-amd64.img',
+      usbTablet: true,
+    }
+    vms.setValue(value);
+
+    vms.save();
+  }) 
+
+  it('Step 2: Check config', () => {
```

Comment:
based on [the practice](https://docs.cypress.io/guides/references/best-practices#Having-tests-rely-on-the-state-of-previous-tests), _steps_ should be put in the same `specify / it` testing.